example/src/main/sandbox/hono-adapter-mqtt-vertx-config.yml

```diff
@@ -9,6 +9,7 @@ hono:
     insecurePortEnabled: true
     keyPath: /run/secrets/mqtt-adapter-key.pem
     certPath: /run/secrets/mqtt-adapter-cert.pem
+    jmsVendorPropsEnabled: true
```

Comment:
For having this working, we need to add JMS vendor properties handling to the MQTT adapter as well (even if the content-type is always octect-stream). Is that right ?

Reply:
Yes, that is right. I just wanted to make sure that the sandbox uses JMS Vendor Properties (if implemented).
I will add support for vendor properties to the MQTT adapter base class as well ...